test/shopify.test.js

```diff
@@ -529,9 +529,158 @@ describe('Shopify', () => {
         expect(res).to.deep.equal(data);
       });
     });
+
+    const addWorkingRESTRequestMock = () => {
+      nock(`https://${shopName}.myshopify.com`)
+        .get('/admin/shop.json')
+        .reply(200, () => ({
+          shop: {
+            name: 'My Cool Test Shop',
+            id: '1'
+          }
+        }));
```

Comment:
```suggestion
        .reply(200, {
          shop: {
            name: 'My Cool Test Shop',
            id: '1'
          }
        });
```

For consistency with other tests, please do not use functions for the second argument of `reply()` .